beacon-chain/core/helpers/committee.go

```diff
@@ -392,20 +393,52 @@ func ClearCache() {
 }
 
 // IsCurrentEpochSyncCommittee returns true if the input public key belongs in the current epoch sync committee along with the sync committee root.
-func IsCurrentEpochSyncCommittee(root [32]byte, pubKey [48]byte) (bool, error) {
+// 1.) Checks if the public key exists in the sync committee cache
+// 2.) If 1 fails, checks if the public key exists in the input current sync committee object
+func IsCurrentEpochSyncCommittee(committee *pb.SyncCommittee, pubKey [48]byte) (bool, error) {
```

Comment:
if we are already passing in the sync committee here, is it worth it to use this method ?